app/components/job-state.js

```diff
@@ -3,7 +3,20 @@ import Ember from 'ember';
 const JobState = Ember.Component.extend({
   classNames: ['job-state', 'dl-horizontal'],
   tagName: 'dl',
-  job: null
+  job: null,
+  liveCpuHours: null,
+  liveVmHours: null,
+  fetchLiveUsage: Ember.on('init', Ember.observer('job', function() {
```

Comment:
If we observe `job`, I suppose this will only fetch once (when the job is set). But If we observed `job.state` (or `job.lastUpdated`, this could re-fetch when the job-watcher notices a change and updates the job.